dist/aurora.js

```diff
@@ -1,6 +1,15 @@
 //[aurora]
 // Uncomment the previous line for testing on webpagetest.org
 
+// Wrap individual metric calls so errors don't break metrics in entire file.
+function runSafely(cb) {
+  try {
+    return cb() || null;
+  } catch {
+    return `ERROR`;
```

Comment:
I'd usually just return null here so the results don't show up in the aggregate analysis. If you want to more clearly indicate that there was an error, maybe for debugging purposes, would including the exception message here be more helpful?

Reply:
Yeah, the "ERROR" was supposed to help me identify which URLs had issues, but including a snapshot of the error message would be even more helpful. Added it!